### PR TITLE
Fixed types in dhcpv4 and dhcpv6

### PR DESCRIPTION
**Pull request by insomniacslk**
Fixes https://github.com/insomniacslk/dhcp/issues/4 .

Used proper types for all the constants defined in the `dhcpv4` and `dhcpv6` packages.

### CODE REVIEW

**Comment by insomniacslk** [comment]
@get9 mind reviewing before I merge?

File: dhcpv6/option_types.go [hunk 1]
```diff
@@ -2,80 +2,79 @@ package dhcpv6
...
+	OPTION_CLIENTID                             OptionCode = 1
+	OPTION_SERVERID                             OptionCode = 2
+	OPTION_IA_NA                                OptionCode = 3
+	OPTION_IA_TA                                OptionCode = 4
+	OPTION_IAADDR                               OptionCode = 5
+	OPTION_ORO                                  OptionCode = 6
+	OPTION_PREFERENCE                           OptionCode = 7
+	OPTION_ELAPSED_TIME                         OptionCode = 8
+	OPTION_RELAY_MSG                            OptionCode = 9
+	_                                                      // skip 10
```
Comment (get9) [inline dhcpv6/option_types.go]: Do you need the `_` here? I think just the comment is sufficient.

**Comment by get9** [comment]
Just one comment, otherwise looks good. Do you mind fixing the BSDP types in the same diff?

File: dhcpv6/option_types.go (hunk 1 above)
Comment (insomniacslk) [reply dhcpv6/option_types.go]: whoops - it's a leftover, good catch!

**Comment by insomniacslk** [comment]
Done, @get9 can you check that out again please?

**Comment by get9** [comment]
Looks good!

**Comment by insomniacslk** [comment]
Merged, thanks!

**Pull request merged**